src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -147,30 +136,55 @@ def __get_events_history(self):
                 f"Error getting events history {e}\n" f"{traceback.format_exc()}"
             )
 
+    @staticmethod
+    def __create_service_info(meta: V1ObjectMeta, kind: str, images: List[str]) -> ServiceInfo:
+        return ServiceInfo(
+            name=meta.name,
+            namespace=meta.namespace,
+            service_type=kind,
+            images=images,
+            labels=meta.labels or {},
+        )
+
     def __discover_services(self):
         try:
```

Comment:
The fact that K8ss python client is faster then Hikaru is knows to us but not clear to the reader. Just add a comment in the K8s client util file, __discover_services and __discover_nodes that says why we use this instead of Hikaru

Reply:
fixing